src/utils/constants.ts

```diff
@@ -3,7 +3,7 @@ import { CHAIN_ID } from 'src/config/chain.d'
 export const APP_ENV = process.env.REACT_APP_ENV
 export const NODE_ENV = process.env.NODE_ENV
 export const IS_PRODUCTION = APP_ENV === 'production'
-export const DEFAULT_CHAIN_ID = IS_PRODUCTION ? CHAIN_ID.ETHEREUM : CHAIN_ID.RINKEBY
+export const DEFAULT_CHAIN_ID = process.env.REACT_DEFAULT_CHAIN_ID || (IS_PRODUCTION ? CHAIN_ID.ETHEREUM : CHAIN_ID.RINKEBY)
```

Comment:
Haven't we already established that the vars should start with REACT_APP in order to work?
Or does it work for you like this?

Reply:
ughh, copy-paste mistake :) thanks for pointing it out!